core/arxiv/submission/tests/examples/test_09_compilation.py

```diff
@@ -0,0 +1,362 @@
+"""Compilation workflow during submission."""
```

Comment:
These are the main tests that were added

Reply:
Tests run fine for me and code appear to exercise compressed and uncompressed limits.